domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/PeriodisertBeregning.kt

```diff
@@ -13,24 +13,56 @@ internal data class PeriodisertBeregning(
     private val fradrag: List<Fradrag>,
     private val fribeløpForEps: Double = 0.0,
 ) : Månedsberegning {
+    private val merknader: Merknader.Beregningsmerknad = Merknader.Beregningsmerknad()
+
     init {
         require(fradrag.all { it.periode == periode }) { "Fradrag må være gjeldende for aktuell måned" }
         require(periode.getAntallMåneder() == 1) { "Månedsberegning kan kun utføres for en enkelt måned" }
+
+        leggTilMerknadVedEndringIGrunnbeløp()
     }
 
     override fun getSumYtelse(): Int = (getSatsbeløp() - getSumFradrag())
         .positiveOrZero()
         .roundToInt()
 
-    override fun getSumFradrag() = fradrag
-        .sumOf { it.månedsbeløp }
-        .limitedUpwardsTo(getSatsbeløp())
+    override fun getSumFradrag() = beregnSumFradrag(fradrag)
 
-    override fun getBenyttetGrunnbeløp(): Int = Grunnbeløp.`1G`.påDato(periode.fraOgMed).toInt()
+    override fun getBenyttetGrunnbeløp(): Int = Grunnbeløp.`1G`.heltallPåDato(periode.fraOgMed)
     override fun getSats(): Sats = sats
     override fun getSatsbeløp(): Double = sats.periodiser(periode).getValue(periode)
     override fun getFradrag(): List<Fradrag> = fradrag
     override fun getFribeløpForEps(): Double = fribeløpForEps
 
+    private fun beregnSumFradrag(fradrag: List<Fradrag>) =
+        fradrag.sumOf { it.månedsbeløp }.limitedUpwardsTo(getSatsbeløp())
+
     override fun equals(other: Any?) = (other as? Månedsberegning)?.let { this.equals(other) } ?: false
+
+    override fun getMerknader(): List<Merknad.Beregning> {
+        return merknader.alle()
+    }
+
+    private fun leggTilMerknadVedEndringIGrunnbeløp() {
+        Grunnbeløp.`1G`.let {
+            if (periode.fraOgMed == it.datoForSisteEndringAvGrunnbeløp(periode.fraOgMed)) {
+                leggTilMerknad(
```

Comment:
Er hensikten men denne merknaden og oppdage tidligere beregninger hvor det har blitt brukt feil grunnbeløp? Og dette bare estimerer vi basert på perioden til beregninga?  Er det ikke bedre å hente det brukte grunnbeløpet fra den persisterte beregninga?

Hva har dette med sosialstønad å gjøre?

Reply:
Egentlig denne bare noe som "ble med" fra branch for virkningstidspunkt (tok merknadene derfra). Lot den bare være da det ikke skader å ta vare på denne informasjonen (f.eks vise til saksbehandler på et tidspunkt).

Ingenting å gjøre med sosialstønad egentlig.